src/controllers/userController.js

```diff
@@ -100,7 +115,23 @@ userController.put('/editUserPassword/:id', async (req, res) => {
         res.status(400).json({ error: error.message });
     }
 });
+//TODO: EDIT USER ROLE ONLY FOR ADMIN
 
 module.exports = {
     userController,
 };
+
+
+
+/*
+Ready user for register
+    "email": "pavel@abv.bg",
+    "organizatorName": "Suzuki Burgas",
```

Comment:
there shouldn't be an organizatorName for regular user

Reply:
fixed